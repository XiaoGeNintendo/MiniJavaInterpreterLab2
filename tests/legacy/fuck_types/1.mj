int main(){
    int x='a';
    char c=65;
    println(x);
    println(c);
    x-=' ';
    c+=32;
    println(x);
    println(c);
    c+=256;
    println(c);
    return 0;
}