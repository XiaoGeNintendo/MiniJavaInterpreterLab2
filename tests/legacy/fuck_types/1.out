97
A
65
a
a
Process exits with the code 0.